images/win/scripts/SoftwareReport/SoftwareReport.Generator.ps1

```diff
@@ -38,6 +38,14 @@ $markdown += New-MDList -Style Unordered -Lines @(
     (Get-NodeVersion)
 )
 
+$markdown += New-MDHeader "Rust packages:" -Level 3
```

Comment:
I believe this section will be better to place after tools section, not on top docs

Reply:
Moved after tools cli